src/bootstrap/build/step.rs

```diff
@@ -300,6 +305,9 @@ impl<'a> Step<'a> {
             Source::ToolRustbook { stage } => {
                 vec![self.librustc(self.compiler(stage))]
             }
+            Source::ToolCargoTest { stage } => {
+                vec![self.librustc(self.compiler(stage))]
```

Comment:
Looks like this doesn't actually depend on the compiler libraries, so I think this can just be `libstd`